### ISSUE TITLE
Excalidraw - Jitsi Helm Chart

### ISSUE THREAD

**Issue by robbie1-uk** [jitsi-contrib/jitsi-helm#124]
Greetings, is there a document about the excalidraw configuration in Jitsi Helm Chart? How is this configuration usually done in Helm?

**Comment by spijet** [comment]
Hello @robbie1-uk!

According to [the `web` image init scripts](https://github.com/jitsi/docker-jitsi-meet/blob/master/web/rootfs/etc/cont-init.d/10-config#L124-L126), the contents of `custom-config.js` file are appended to the end of the rendered `/config/config.js` file. You can attach to a running `web` pod and check if it actually happens by reading the last N lines of this config file. Another option would be using the Developer Console in your browser and checking the `config.js` file your browser gets when connecting to your Jitsi Meet installation.

**Comment by robbie1-uk** [comment]
hello @spijet, thank you
So should I make these changes in the running pod? Because when I write in values.yaml, it does not make the changes.

**Comment by spijet** [comment]
No, I meant that you can check if the configuration is there by connecting to a pod and reading the contents of `custom-config.js`. If you fill the release values correctly, your custom settings should appear at the end of that file.
